command/command.go

```diff
@@ -1,215 +1,20 @@
 package command
 
 import (
-	"encoding/json"
-	"fmt"
-	"os"
-	"reflect"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/smallstep/cli/crypto/fingerprint"
-	"github.com/smallstep/cli/usage"
 	"github.com/urfave/cli"
-	"go.step.sm/cli-utils/step"
 )
 
-// IgnoreEnvVar is a value added to a flag EnvVar to avoid the use of
-// environment variables or configuration files.
-const IgnoreEnvVar = "STEP_IGNORE_ENV_VAR"
-
-var cmds []cli.Command
-var currentContext *cli.Context
-
-func init() {
-	os.Unsetenv(IgnoreEnvVar)
-	cmds = []cli.Command{
-		usage.HelpCommand(),
-	}
-}
-
-// Register adds the given command to the global list of commands.
-// It sets recursively the command Flags environment variables.
-func Register(c cli.Command) {
-	setEnvVar(&c)
-	cmds = append(cmds, c)
-}
-
-// Retrieve returns all commands
-func Retrieve() []cli.Command {
-	return cmds
-}
-
 // ActionFunc returns a cli.ActionFunc that stores the context.
 func ActionFunc(fn cli.ActionFunc) cli.ActionFunc {
 	return func(ctx *cli.Context) error {
-		currentContext = ctx
```

Comment:
Why not use the ActionFunc in cli-utils?

Reply:
This `ActionFunc` was still in use by two commands, so I left that as is. It seems to be the same function; just not refactored when the function was extracted to `cli-utils`, so I can probably clean this out too.

In general, it's a matter of discoverability, clarity and remembering the existence of these internal, shared libraries that affect whether I'll use them or not. It's easier to find code in the same repository, than when having to look for them in dependencies.